examples/jss/src/components/content/content.js

```diff
@@ -0,0 +1,39 @@
+import React, { PropTypes } from 'react';
+import useSheet from 'react-jss';
+
+const styles = {
+  text: {
+    fontSize: '1.25rem',
+    fontWeight: '300',
+    lineHeight: '1.5em',
+    margin: '0',
+    padding: '.65625rem 0 .98438rem',
+  },
+  media: {
+    borderRadius: '.35rem',
+    border: '1px solid #e1e8ed',
+    color: '#1da1f2',
+    display: 'block',
+    margin: '.65625rem 0 1.3125rem',
+  },
+  image: {
+    display: 'block',
+    maxWidth: '100%',
+  },
+};
+
+const Content = ({ text, media, sheet }) => (
```

Comment:
same here, looks like much less repition to me if you spread the sheet and just pick classes ... you rarely need the sheet ref:
```
+const Content = ({ text, media, sheet: {classes} }) => (
```